deployments/mumbai/usdc/deploy.ts

```diff
@@ -23,40 +23,40 @@ async function deployContracts(deploymentManager: DeploymentManager, deploySpec:
   const ethers = deploymentManager.hre.ethers;
   const signer = await deploymentManager.getSigner();
 
+  // https://docs.polygon.technology/docs/develop/l1-l2-communication/fx-portal/#contract-addresses
+  const FX_CHILD = "0xCf73231F28B7331BBe3124B907840A94851f9f11";
+
   // Deploy PolygonBridgeReceiver
-  const polygonBridgeReceiver = await deploymentManager.deploy(
-    'polygonBridgeReceiver',
+  const bridgeReceiver = await deploymentManager.deploy(
+    'bridgeReceiver',
     'bridges/polygon/PolygonBridgeReceiver.sol',
     [
+      FX_CHILD,
       signer.address, // admin
     ]
   );
 
   // Deploy L2 Timelock
   const l2Timelock = await deploymentManager.deploy(
-    'l2Timelock',
+    'timelock',
     'vendor/Timelock.sol',
     [
-      polygonBridgeReceiver.address, // admin
+      bridgeReceiver.address,        // admin
       2 * secondsPerDay,             // delay
       14 * secondsPerDay,            // grace period
       2 * secondsPerDay,             // minimum delay
       30 * secondsPerDay             // maxiumum delay
     ]
   );
 
-  // https://docs.polygon.technology/docs/develop/l1-l2-communication/fx-portal/#contract-addresses
-  const FX_CHILD = "0xCf73231F28B7331BBe3124B907840A94851f9f11"; //
-  const MAINNET_TIMELOCK = "0x6d903f6003cca6255d85cca4d3b5e5146dc33925";
-
-  // Initialize PolygonBridgeReceiver
-  trace(`Initializing PolygonBridgeReceiver`);
-  await polygonBridgeReceiver.initialize(
-    MAINNET_TIMELOCK,       // mainnet timelock
-    l2Timelock.address,     // l2 timelock
-    FX_CHILD                // fxChild
+  const GOERLI_TIMELOCK = "0x339B2D3bf0406DF82f8fa7B0d855a3f47562d8D7";
```

Comment:
Would be neat if we also refactored `deploy` to have multiple deployment managers so we can pull this address dynamically.

Reply:
how would you specify what the second deployment manager should be? via command line maybe:

```
yarn hardhat deploy --network mumbai --deployment usdc --parentNetwork goerli --parentDeployment usdc
```

Reply:
Hmm good point. Might be a bit too complicated to pass it in via command line, so maybe we should just keep it simple for now and leave that to a future improvement.